lib/interfaces/heroku_interface.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../http_post_interface'
+Dir['./lib/slack_commands/*.rb'].each { |file| require file }
+
+class HerokuInterface < HttpPostInterface
+  def handle_heroku_webhook(payload)
+    params = Rack::Utils.parse_nested_query(payload)
+    user = params['user'].split('@')[0]
+    release = "#{params['release']} " if params['release']
+    server = params['url'].split('/').last.split('.').first[8..-1]
```

Comment:
Good ideas. I definitely breezed over the logic for this. Definitely agree about being explicit about `joyable` in the extraction process.